spring-kafka/src/main/java/org/springframework/kafka/annotation/KafkaBootstrapConfiguration.java

```diff
@@ -53,4 +65,27 @@ public KafkaListenerEndpointRegistry defaultKafkaListenerEndpointRegistry() {
 		return new KafkaListenerEndpointRegistry();
 	}
 
+	static class KStreamsRegistrar implements ImportBeanDefinitionRegistrar, BeanClassLoaderAware {
+
+		private ClassLoader classLoader;
+
+		@Override
+		public void setBeanClassLoader(ClassLoader classLoader) {
+			this.classLoader = classLoader;
+		}
+
+		@Override
+		public void registerBeanDefinitions(AnnotationMetadata importingClassMetadata,
+				BeanDefinitionRegistry registry) {
+			if (ClassUtils.isPresent(org.apache.kafka.streams.kstream.KStreamBuilder.class.getName(),
```

Comment:
I suppose we should use the class name as a String, or else this would blow up if the class is not on the classpath?